examples/child-design-system/src/styles/base/_override.typography.scss

```diff
@@ -6,5 +6,5 @@
 .ds-h4,
 .ds-h5,
 .ds-h6 {
```

Comment:
I swapped out oswald for lato because it was more useable and realistic as a body font. I also added these typography overrides so we actually used both new fonts

Reply:
Oh ha! I was going to use Oswald for the headings, thanks for adding those!